plugins/font/plugin.js

```diff
@@ -94,7 +124,7 @@
 				this.startGroup( lang.panelTitle );
 
 				// Add `(Default)` item as a first element on the drop-down list.
-				this.add( this.defaultValue, defaultText, defaultText );
+				this.add( '', defaultText, defaultText );
```

Comment:
`this.defaultValue` actually tells more than a magic empty string, so maybe it's worth to keep it as named property? Also easier to update value later if needed.